pkg/sql/logictest/testdata/logic_test/schema

```diff
@@ -1481,3 +1481,24 @@ SELECT qux();
 NULL
 
 subtest end
+
+subtest empty_schema_name
+
+user root
+
+statement ok
+set role root;
+use defaultdb;
+
+skipif config local-legacy-schema-changer
+statement error pgcode 42601 pq: empty schema name
+CREATE SCHEMA ""."";
+
+onlyif config local-legacy-schema-changer
+statement error pgcode 42601 pq: empty database name
```

Comment:
looks like a different error. i'm a little surprised this differs from `master`; do you know what changed?

```
        /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test_/local-legacy-schema-changer_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/schema:1499: 
        expected:
        pq: empty database name
        got:
        pq: database "" does not exist
```

Reply:
Its linked to https://github.com/cockroachdb/cockroach/pull/134334 on master. I'll update the expected errors for the backports.
<!-- Reviewable comment -OCJ4RamCdCoBMVEhNDU:bxbxyae -->
<!-- Sent from Reviewable.io -->